### PR TITLE
remove v1 monitoring code

### LINKED ISSUE #10490: Remove v1 monitoring embedded pages
We embed v1 monitoring in a few places when v1 monitoring is installed:

- Cluster explorer dashboard page - see: https://github.com/rancher/dashboard/blob/master/shell/pages/c/_cluster/explorer/index.vue#L570
- Node detail page - see: https://github.com/rancher/dashboard/blob/master/shell/detail/node.vue#L330
- Pod detail page - see: https://github.com/rancher/dashboard/blob/master/shell/detail/pod.vue#L262
- Workload detail page: see: https://github.com/rancher/dashboard/blob/master/shell/detail/workload/index.vue#L456

v1 monitoring has been removed, so we should remove this code to clean up the codebase.

Note: It looks like we've removed all v1 monitoring charts, so I can't find a way to install it any longer - so for this ticket, we will remove the code and verify through code review that the appropriate code has been removed - since it does not look like we can test v1 monitoring any longer. 

Note: As part of doing this, search the code for `haveV1Monitoring` and remove/claan-up areas associated with this.

Also check doc links if still valid: perviously reported that some non-working doc links found in 2 files (shell/chart/monitoring/steps/uninstall-v1.vue & shell/pages/c/_cluster/explorer/tools/index.vue).

### PR DESCRIPTION
**Pull request by aalves08**
### Summary
Fixes #10490 

### Occurred changes and/or fixed issues

- remove v1 monitoring code
- add e2e tests

### Technical notes summary

- Nothing noteworthy

### Areas or cases that should be tested

- make sure `node` detail page works - Added simple assertion to  https://github.com/rancher/dashboard/blob/master/cypress/e2e/tests/pages/explorer/node-list.spec.ts ✅ 
- make sure `pod` detail page works - already covered in  https://github.com/rancher/dashboard/blob/master/cypress/e2e/tests/pages/explorer/workloads/pods.spec.ts ✅ 
- make sure `workload` detail page works - already covered in  https://github.com/rancher/dashboard/blob/master/cypress/e2e/tests/pages/explorer/workloads/workloads.spec.ts ✅ 
- make sure `cluster explorer` page works - already covered by https://github.com/rancher/dashboard/blob/master/cypress/e2e/tests/pages/explorer/cluster-dashboard.spec.ts ✅ 
- make sure `c/cluster/monitoring` page works
- make sure `v2 monitoring` works

### Areas which could experience regressions

Make sure we cover the areas mentioned in the previous points to avoid regressions. Manual testing might be needed to ensure V2 monitoring is working fine

### Screenshot/Video

https://github.com/rancher/dashboard/assets/97888974/7464773f-4cd9-47e8-ab8e-8dc4bb63f1ce

### Checklist
- [x] The PR is linked to an issue and the linked issue has a Milestone, or no issue is needed
- [x] The PR has a Milestone 
- [x] The PR template has been filled out
- [x] The PR has been self reviewed 
- [x] The PR has a reviewer assigned
- [x] The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
- [x] The PR has reviewed with UX and tested in light and dark mode, or there are no UX changes

### CODE REVIEW

**Comment by aalves08** [comment]
@nwmac @richard-cox added you both as reviewers since this need a keen eye. I went to each of those views and all looked good. 
Not fully sure if the `uninstall` can all go away, but I believe I captured the correct code removal. Didn't test this... Is it testable even?

**Review by richard-cox — CHANGES_REQUESTED** [review]
There's a couple of items in the checklist that still need doing

```
The PR template has been filled out
The PR has automated tests or clear instructions for manual tests and the linked issue has appropriate QA labels, or tests are not needed
```

Automated test side, we probably need some to cover below. Hopefully all have existing tests though
- Node list --> click on one --> confirm something trivial on page
- Workload list --> click on one --> confirm something trivial on page
- Cluster Dashboard --> confirm something trivial on page

Monitoring is a bit more of a pain, would need some manual tests for that (it's a honker to install on the small image used by gh ci)

**Comment by aalves08** [comment]
@richard-cox the issue has `QA/None` label... Should I change it to `QA/manual-test`?

**Review by richard-cox — CHANGES_REQUESTED** [review]

File: cypress/e2e/po/components/sortable-table.po.ts [hunk 1]
```diff
@@ -92,6 +92,11 @@ export default class SortableTablePo extends ComponentPo {
     return this.self().contains('tbody tr', name);
   }
 
+  rowElementLinkWithIndex(index: number) {
+    // return this.self().contains(`tbody tr:nth-child(${ index })`);
```
[the remark this answers is not in the document]
Comment (aalves08) [reply cypress/e2e/po/components/sortable-table.po.ts:96]: Removed. Thanks

**Review by richard-cox — APPROVED** [review]

**Pull request merged**